templates/client-daemonset.yaml

```diff
@@ -163,7 +169,17 @@ spec:
               hostPort: 8502
               name: grpc
             - containerPort: 8301
-              name: serflan
+              {{- if .Values.client.exposeGossipPorts }}
+              hostPort: 8301
+              {{- end }}
+              protocol: "TCP"
+              name: serflan-tcp
+            - containerPort: 8301
+              {{- if .Values.client.exposeGossipPorts }}
+              hostPort: 8301
+              {{- end }}
+              protocol: "UDP"
+              name: serflan-udp
```

Comment:
This is a quirk of defining `hostPorts` because each <hostIP, hostPort, protocol> combination must be unique. It shouldn't be necessary for the servers, though we could if we wanted to be explicit about the protocols being used.

Reply:
I was thinking for consistency as well